config/info.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// YorcVersion is the used version of YORC
+var YorcVersion = "Must Be defined by MakeFile"
```

Comment:
Hum not sure that the `config` package is the right place for this. `config` is for things that are configurable...

Reply:
I didn't find any better fitting package for this. `config` looked liked the the least worst. 
So maybe a new one must be created ? Maybe `info` or `others`

Reply:
And what about a `server/info` package for info about a server? 
@ystia/yorc-dev WDYT?

Reply:
Yep